src/components/Sidebar.tsx

```diff
@@ -0,0 +1,54 @@
+import { FiUsers, FiLogOut, FiFileText } from 'react-icons/fi';
+
+const Sidebar = () => {
+    return (
+        <div 
+            className="w-16 h-screen flex flex-col justify-between py-4"
+            style={{ backgroundColor: '#002D7E' }}
+        >
+            <div>
+                {/* Square with rounded edges containing the "K" */}
+                <div className="flex justify-center mt-4 mb-8">
+                    <div className="bg-[#DEF2FF] rounded-lg w-12 h-12 flex items-center justify-center">
```

Comment:
create predefined colors in the tailwind theme so we can use bg-primary or pg-secondary, etc